<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComAnalysis</class>
 <widget class="QDialog" name="ComAnalysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1205</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/photoes/cgjt.png</normaloff>:/photoes/cgjt.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>1100</width>
       <height>1000</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; color:#000000;&quot;&gt;隔离开关操作机构扭矩及电流监测综合分析&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>标准波形显示</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="chkYangShowTor">
               <property name="text">
                <string>扭矩</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="chkYangShowAng">
               <property name="text">
                <string>角度</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="chkYangShowCur">
               <property name="text">
                <string>电流</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>当前波形显示</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="chkCurShowTor">
               <property name="text">
                <string>扭矩</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="chkCurShowAng">
               <property name="text">
                <string>角度</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="chkCurShowCur">
               <property name="text">
                <string>电流</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="btnStarTorComparea">
              <property name="text">
               <string>扭矩波形比对</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnStopTorCompare">
              <property name="text">
               <string>清除扭矩比对数据</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnStarCurComparea">
              <property name="text">
               <string>电流波形比对</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnStopCurCompare">
              <property name="text">
               <string>清除电流对比数据</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>AcadEref</family>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;结论：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="textEditConclusion">
              <property name="html">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QChartView" name="chartViewAnalysis" native="true">
          <property name="minimumSize">
           <size>
            <width>800</width>
            <height>500</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTableWidget" name="tableWidgetWaveFile">
          <column>
           <property name="text">
            <string>序号</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>文件名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>分/合闸</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>选择</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QPushButton" name="btnDataExport">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>波形数据导出</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDataImport">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>波形数据导入</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDataDelete">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>波形数据删除</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSaveToYB">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>保存为标准波形</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
